packages/onboarding/src/select-items-alt/index.tsx

```diff
@@ -2,32 +2,40 @@ import { Button } from '@automattic/components';
 import { Tooltip } from '@wordpress/components';
 import { Icon, info } from '@wordpress/icons';
 import classnames from 'classnames';
+import { TranslateResult } from 'i18n-calypso';
 import React from 'react';
 import './style.scss';
-import { SelectAltItem } from '../../types';
+
+export interface SelectItemAlt< T > {
+	show: boolean;
+	key: string;
+	description: TranslateResult;
+	value: T;
+	actionText: TranslateResult;
+	disable: boolean;
+	disableText: TranslateResult;
+}
```

Comment:
Naming is better this way, but search for "SelectAltItem" in the codebase, you'll find a couple places where it still needs to be changed! (for example `client/signup/steps/intent/intents.tsx`)

Reply:
Oof, nice catch. Updated the two instances I saw.